### PR TITLE
Moved Katib documentation from kubeflow/kubeflow

### PR DESCRIPTION
**Pull request by amsaha**
Fixes Issue #72.

### CODE REVIEW

File: content/docs/guides/components/hyperparameter.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Deploying Katib
+
+[Katib](https://github.com/kubeflow/katib) is a hyperparameter tuning framework, inspired by
+[Google Vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf).
+
+To deploy katib,
+```shell
+ks pkg install kubeflow/katib@master
+ks generate katib katib
+ks apply YOUR_ENV -c katib
```
Comment (jlewi) [inline content/docs/guides/components/hyperparameter.md]: nit: YOUR_ENV -> ${ENV}
This way its a proper bash command

Comment (amsaha) [reply content/docs/guides/components/hyperparameter.md]: Sure. I simply copy pasted what is there at: https://github.com/kubeflow/kubeflow/blob/master/kubeflow/katib/README.md.

**Comment by amsaha** [comment]
@jlewi Took care of your suggestion. Please review.

**Pull request merged**